pedalboard/io/ReadableAudioFile.h

```diff
@@ -281,7 +281,12 @@ class ReadableAudioFile
         for (int c = 0; c < numChannels; c++) {
           juce::FloatVectorOperations::convertFixedToFloat(
               channelPointers[c], (const int *)channelPointers[c], scaleFactor,
+#ifdef LINUXX
+              static_cast<size_t>(numSamples));
```

Comment:
I was trying to use the latest version of JUCE 7.0.1 which also has a size_t signature. Probably a little premature doing this.
I could change this line to 
`static_cast<int>(numSamples));`
which would mean I don't need the the preprocessor check for LINUX and this will also work for JUCE 7.x
Is that preferable? I am not keen on changing the type coming in to this function as that could have unforeseen side effects, but I could try it if you want.